### PR TITLE
user sign-up

### PR DESCRIPTION
**Pull request by ngothaiminh**

### CODE REVIEW

File: app/views/devise/registrations/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      </div>
+
+      <div class="field">
+        <%= f.label :password_confirmation %><br />
+        <%= f.password_field :password_confirmation, autocomplete: "off",
+          class: "form-control" %>
+      </div>
+
+      <div class="actions text-center">
+        <%= f.submit "Sign up", class: "btn btn-primary submit-btn" %>
```
Comment (vuducluan) [inline app/views/devise/registrations/new.html.erb]: I18n

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,5 +1,15 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.for(:sign_up) { |u| u.permit(:name, :email,
```
Comment (dieunb) [inline app/controllers/application_controller.rb]: if your line code is too long, > 80 chars, should use `do..end`

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+      :password, :password_confirmation, :remember_me) }
+    devise_parameter_sanitizer.for(:sign_in) { |u| u.permit(:login, :email,
```
Comment (dieunb) [inline app/controllers/application_controller.rb]: `do..end`

File: app/controllers/application_controller.rb [hunk 4]
```diff
@@ -1,5 +1,15 @@
...
+      :password, :remember_me) }
+    devise_parameter_sanitizer.for(:account_update) { |u| u.permit(:name,
```
Comment (dieunb) [inline app/controllers/application_controller.rb]: `do..end`

File: config/initializers/devise.rb [hunk 5]
```diff
@@ -0,0 +1,23 @@
+Devise.setup do |config|
+  config.mailer_sender = "please-change-me-at-config-initializers-devise@example.com"
```
Comment (dieunb) [inline config/initializers/devise.rb]: don't remove all comments in this file

**Pull request merged**